Formula/grumpy.rb

```diff
@@ -0,0 +1,31 @@
+class Grumpy < Formula
+  desc "Python to Go source code transcompiler and runtime"
+  homepage "https://github.com/google/grumpy"
+  head "https://github.com/google/grumpy.git"
+
+  depends_on "go"
+
+  def install
+    system "make"
+    system "make", "DESTDIR=#{prefix}", "install"
+    bin.install "#{prefix}/usr/bin/grumpc"
+    bin.install "#{prefix}/usr/bin/grumprun"
+  end
+
+  def caveats
+    <<-EOS.undent
+      NOTE:  Before using grumpc and/or grumprun, you MUST:
+      export GOPATH="#{prefix}/usr/lib/go"
+      export PYTHONPATH="#{prefix}/Library/Python/2.7/site-packages"
+
+      Then you can test out grumpy with:
+      echo "print 'hello, grumpy world'" | grumprun
+    EOS
+  end
+
+  test do
+    ENV["GOPATH"] = #{prefix}/usr/lib/go
+    ENV["PYTHONPATH"] = #{prefix}/Library/Python/2.7/site-packages
```

Comment:
Sorry, I did mention this the first time, but the right side of these lines need to be quoted:
```
    ENV["GOPATH"] = "#{prefix}/usr/lib/go"
    ENV["PYTHONPATH"] = "#{prefix}/Library/Python/2.7/site-packages"
```